src/quo/components/controls/view.cljs

```diff
@@ -12,37 +12,47 @@
 (defn control-builder
   [component]
   (fn [props]
-    (let [{:keys [value onChange disabled]}
+    (let [{:keys [
+                  ;                   value
+                  onChange disabled]}
           (bean/bean props)
-          state (animated/use-value 0)
-          tap-state (animated/use-value (:undetermined gh/states))
+          state 0
+          tap-state (:undetermined gh/states)
           tap-handler (animated/on-gesture {:state tap-state})
           hold (react/use-memo
                 (fn []
-                  (animated/with-timing-transition
-                   (animated/eq tap-state (:began gh/states))
-                   {}))
+                    ;commented out to upgrade react-native-reanimated to v3 and react-native to 0.72
+                    ;TODO: replace this with an updated implementation
```

Comment:
This is old code, why do we need to keep the code commented out? I'd say if it's old implementation such as this just get rid of animations if it's affecting the upgrade. I don't see why we would care about animations in old screens, they would still be functional, which is all we need until we migrate them.